src/Markup/Xamarin.CommunityToolkit.Markup/Xamarin.CommunityToolkit.Markup.csproj

```diff
@@ -33,8 +33,8 @@
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Forms" Version="5.0.0.2125" />
-    <PackageReference Include="mdoc" Version="5.8.3" PrivateAssets="all" />
+    <PackageReference Include="Xamarin.Forms" Version="5.0.0.2291" />
```

Comment:
Let's not update this NuGet Package unless we have to, because it'll force all Xamarin.CommunityToolkit.Markup users to then upgrade to the latest version Xamarin.Forms

Reply:
as far as I remember the last time we updated it was because we want the users to update to the latest XF version.
Also, this version has fixes for our Sample app

Reply:
You updated it on September: ![image](https://user-images.githubusercontent.com/33021114/146601283-7b814c32-90f4-46f8-9a61-8cb195276a16.png)

the new version contains fix: https://github.com/xamarin/Xamarin.Forms/pull/7728